classes/lang/KeysReference/FeatureFlagLang.php

```diff
@@ -42,3 +42,7 @@
 // Legacy link feature flags
 trans('Attribute groups', 'Admin.Advparameters.Feature');
 trans('Enable or disable the migrated attribute groups page.', 'Admin.Advparameters.Help');
+
+// Authorization server feature flag
+trans('Authorization server', 'Admin.Advparameters.Feature');
+trans('Enable or disable the authorization server page.', 'Admin.Advparameters.Help');
```

Comment:
Hello, **"Enable or disable the authorization server page."** Is this a toggle button? If yes, is its default position "off"? Thanks :-)

Reply:
Hi, yes off by default :)